README.md

```diff
@@ -43,22 +43,22 @@ composer create-project --no-install contao/managed-edition <directory> <branch>
 ```
 
 Replace `<directory>` with the directory you want to install the Managed
-Edition in (use `.` for the current one). Replace `<branch>` with `dev-master`
-if you want to add a new feature or with `<lts-version>.x-dev` (currently
-`4.9.x-dev`) if you want to fix a bug.
+Edition in (use `.` for the current one). Replace `<branch>` with `4.x` if you
```

Comment:
shouldn't that be `4.x-dev`?

Reply:
Yes. Fixed in be86e327d6e30ac02ab02387c3bd94463c501e6c.